python/deepgram-transcribe-audio/README.md

```diff
@@ -0,0 +1,95 @@
+# Transcribe Audio Files
+
+A Python Cloud Function that transcribes an Audio file with the help of Deepgram.
+
+_Example input 1:_
+
+```json
+{
+    "fileUrl": "https://static.deepgram.com/examples/interview_speech-analytics.wav"
+}
+```
+
+_Example output 1:_
+
+```json
+{
+    "response": {
+        "deepgramData": {
+            "confidence": 0.9926758,
+            "transcript": "another big problem in the ... company succeed using this product",
+            "words": [
+                {
+                    "confidence": 0.9995117,
+                    "end": 0.839599,
+                    "start": 0.33959904,
+                    "word": "another"
+                },
+				...
+                {
+                    "confidence": 0.99365234,
+                    "end": 95.770004,
+                    "start": 95.41,
+                    "word": "product"
+                }
+            ]
+        },
+        "success": true
+}
+```
+
+_Example input 2:_
+
+```json
+{
+
+    "fileUrl": "WRONG_URL"
+}
+```
+
+_Example output 2:_
+
+```json
+{
+    "response": {
+        "message": "Please provide a valid audio URL",
+        "success": false
+    }
+}
```

Comment:
```suggestion
{
    "message": "Please provide a valid audio URL",
    "success": false
}
```

I would suggest removing the open runtimes response structure and only focus on the response of our function.

Let's please apply this change to all the other output examples.